app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/commontask/components/CommonTaskAppBar.kt

```diff
@@ -94,6 +99,21 @@ fun CommonTaskAppBar(
                     expanded = isMenuExpanded,
                     onDismissRequest = { isMenuExpanded = false }
                 ) {
+                    // Copy link
+                    DropdownMenuItem(
+                        onClick = {
+                            isMenuExpanded = false
+                            editActions.setCopyLinkInClipboard()
+                            Toast.makeText(context, copyText, Toast.LENGTH_SHORT).show()
```

Comment:
Toast is a bit outdated. There is Snackbar which is used here to display error messages. I think you should use it instead